plugins/cpu/fu-cpu-device.c

```diff
@@ -95,12 +108,34 @@ fu_cpu_device_convert_vendor (const gchar *vendor)
 	return vendor;
 }
 
+static gboolean
+fu_cpu_is_updatable (GError** error)
+{
+	if (g_access ("/lib/firmware/intel-ucode", R_OK|W_OK) < 0)
```

Comment:
Would it make more sense to check /lib/firmware is writable, and if it's not, then check if you're intel if that path works?